spec/linter-eslint-spec.js

```diff
@@ -4,6 +4,7 @@ import * as path from 'path'
 import * as fs from 'fs'
 import { tmpdir } from 'os'
 import rimraf from 'rimraf'
+// eslint-disable-next-line import/no-named-as-default, import/no-named-as-default-member
```

Comment:
I'm guessing this line snuck in from the `eslint-plugin-import@2.0.0` part, it's not necessary (yet), so it should be removed.